cmd/run.go

```diff
@@ -58,9 +62,12 @@ func init() {
 	runCmd.Flags().StringVarP(&sshKeyPath, "ssh-key-path", "i", "", "path to private ssh key")
 	runCmd.Flags().UintVarP(&sshTimeout, "ssh-connect-timeout", "c", 30, "time to wait (in secs) for hosts to accept connection")
 	runCmd.Flags().UintVarP(&timeout, "timeout", "t", 300, "total time to wait (in secs) for host processing to complete")
+	runCmd.Flags().StringSliceVarP(&certIPs, "cert-ips", "", []string{}, "list of ips in ca cert (comma-delimited)")
```

Comment:
I would add shorthand letters to these flags to keep the design uniform, like `-d` for `certDNSNames` and `-a` for `certIPs`, or use the func `StringSliceVar()` instead.

Reply:
Added